bin/faucet/src/server/get_tokens.rs

```diff
@@ -141,6 +153,22 @@ impl RawMintRequest {
             .validate(self.asset_amount)
             .ok_or(InvalidRequest::AssetAmount(self.asset_amount))?;
 
+        // Check the server timestamp
+        if !check_server_timestamp(self.server_timestamp) {
+            return Err(InvalidRequest::ExpiredServerTimestamp);
+        }
+
+        // Check the server signature
+        if !check_server_signature(&self.server_signature, &self.pow_seed, self.server_timestamp) {
+            error!("[get_tokens] invalid server signature");
```

Comment:
Correct me if I'm wrong but this could be caused by bad requests right? In which case we wouldn't want to log error.

Reply:
Yes, basically we are checking that the user is not sending a solution of a seed that wasn;'t created by the server.

Reply:
Should we change `error!` to warn or debug then?

Reply:
It was already removed, since this is not an application error

Reply:
Ah gotcha - was confused!